libnmstate/schema.py

```diff
@@ -117,6 +118,7 @@ class InterfaceType(object):
         OVS_INTERFACE,
         VLAN,
         VXLAN,
+        TEAM,
```

Comment:
IMHO it woudl be nicer to  keep this list sorted.

Reply:
Sure!